Makefile

```diff
@@ -33,7 +33,14 @@ release-tests:
 
 #This is used on CircleCI because the Redis Docker container is already started unlike the local tests command
 ci-tests:
-	@$(REBAR3) ct --dir="test,test-redis" --logdir logs/ct
+	@if [ "$(ERL_VERSION)" -ge "25" ]; then\
```

Comment:
Here things get ugly. Need two different versions of meck to keep things working.